codalab/lib/upload_manager.py

```diff
@@ -13,11 +13,14 @@ class UploadManager(object):
     """
 
     def __init__(self, bundle_model, bundle_store):
+        from codalab.lib import zip_util
```

Comment:
question - why does making imported module a variable can accelerate running? Since the importing also happens when the constructor is called, seems like the importing isn't delayed too much.

Reply:
When it's moved to the constructor, it's only called when someone calls something like `cl upload`, which initializes `UploadManager`. It won't be called if someone makes another command such as `cl --help` or `cl work`, though.